TUM Campus App/BookRental.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BookRental.swift
+//  TUM Campus App
+//
+//  Created by Tim Gymnich on 19.04.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import Foundation
+import Kanna
+
+class BookRental: DataElement {
+    
+    let title: String
+    let author: String
+    let id: String
+    let deadline: String
+    let bib: String
+    let prolong: ProlongPossibilty
+    
+    enum ProlongPossibilty: String {
+        
+        case notYetPossible = "Eine Verlängerung ist noch nicht möglich."
+        case possible = "?" //we still need to find out this string
+        case notPossible = "??" //we still need to find out this string
+    }
+    
+    init(title: String, author: String, id: String, deadline: String, bib: String, prolong: ProlongPossibilty) {
+        self.title = title
+        self.author = author
+        self.id = id
+        self.deadline = deadline
+        self.bib = bib
+        self.prolong = prolong
+    }
+    
+    func getCellIdentifier() -> String {
+        return "bookRental"
+    }
+    
+    var text: String {
+        return title
+    }
+}
+
+extension BookRental: CardDisplayable {
+    
+    var cardKey: CardKey {
+        return .bookRental
+    }
+}
+
+extension BookRental {
+    
+    convenience init?(td1: XMLElement, td2: XMLElement) {
+        
+        guard let title = td1.at_css("strong")?.content,
```

Comment:
I know it's not really MVC. But neither is iOS really. I was thinking of moving this to another file but in general, failable initializers make our work much easier and cleaner